test/tchannel.js

```diff
@@ -0,0 +1,169 @@
+var test = require('tape');
+var TChannel = require('../index.js');
+
+var serverOptions = {host: '127.0.0.1', port: 4040, listening: false},
+  clientOptions = {host: '127.0.0.1', port: 4041, listening: false},
```

Comment:
style nitpicks:

For consistency it would be great if we had:

 - one `var` statement per variable
 - four spaces or tabs.